### PR TITLE
feat(e2e): Manage User Settings e2e tests

### PR DESCRIPTION
**Pull request by yasserfaraazkhan**
e2e test case for https://github.com/mattermost/mattermost/pull/27583

test-management : https://github.com/mattermost/mattermost-test-management/pull/174

```release-note
NONE
```

### CODE REVIEW

**Review by marianunez — COMMENTED** [review]
@yasserfaraazkhan overall looks good! Could we add tests for the license requirement (with and without license)?

**Review by harshilsharma63 — APPROVED** [review]

**Comment by harshilsharma63** [comment]
@yasserfaraazkhan can we please include one more test where the server is licensed but the user doesn't have write permission for user management and so doesn't see the button?

File: e2e-tests/cypress/tests/integration/channels/system_console/user_management_spec.js [hunk 1]
```diff
@@ -233,6 +233,23 @@ describe('User Management', () => {
         cy.url().should('include', '/admin_console/about/license');
     });
 
+    it('Admin cannot access Manage User Settings option on a unlicensed instance', () => {
+        // # Login as sysadmin.
+        cy.apiLogin(sysadmin);
+        cy.visit('/admin_console/user_management/users');
+        cy.intercept('**api/v4/reports/users?**').as('getUserList');
```
Comment (hmhealey) [inline e2e-tests/cypress/tests/integration/channels/system_console/user_management_spec.js:240]: Interesting. I haven't seen this method before. That seems really helpful

**Review by hmhealey — APPROVED** [review]

**Pull request merged**